metricbeat/module/elasticsearch/test_elasticsearch.py

```diff
@@ -74,6 +74,7 @@ def test_metricsets(self, metricset):
         self.check_metricset("elasticsearch", metricset, self.get_hosts(), self.FIELDS +
                              ["service"], extras={"index_recovery.active_only": "false"})
 
+    @unittest.skipUnless(metricbeat.INTEGRATION_TESTS, "integration test")
```

Comment:
Umm, isn't it failing in master?

Reply:
It probably should be failing in master. Perhaps we don't run these tests outside of the INTEGRATION_TEST=true environment any where on CI.